src/org/labkey/targetedms/parser/PeakAreaRatioCalculator.java

```diff
@@ -72,6 +72,11 @@ public void init(Map<SkylineDocImporter.SampleFileKey, SampleFile> skylineIdSamp
 
             for(Transition transition: precursor.getTransitionsList())
             {
+                // Issue 41788: Exclude non-quantitative transitions from peak area ratio calculations
+                if(!transition.isQuantitativeTransition())
```

Comment:
When calculating total areas, we also skip over all of the MS2 transitions if the Transition Settings Full Scan Acquisition Method was "DDA".
In Skyline, this logic is taken care of in the method "bool TransitionDocNode.IsQuantitative(SrmSettings settings)".
The same sort of check should probably be happening here.

Reply:
A few of questions about bool TransitionDocNode.IsQuantitative(SrmSettings settings)
- Does IsMs1() apply only to proteomic transitions? 
- ComplexFragmentIon.IsMS1() is looking at TransitionLosses.  We only save transition losses for proteomic transitions.  Should we be saving them for small molecule transitions as well? Should we move the variable from Transition to GeneralTransition?
- I don't think we are saving the children of ComplexFragmentIon so we will not be able to do this check: Children.Values.All(child => child.IsMs1

I will update the test with the document used in Skyline's Quantitative test.  If there is a better one let me know. 
Thanks!

Reply:
For small molecules, a transition is MS1 if and only if its fragment ion is precursor.

For proteomic transitions, we also need to look at the neutral losses and complex fragment ion.

I made a trial implementation of this here:
https://github.com/nickshulman/targetedms/commit/c5639547b838239ad82f04d8f695bd9f403c20ef
(Haven't tested it, but it probably works)

Reply:
Thanks for the looksLikeIntactPrecursor() method!  Do you have a Skyline document I can test this with?